docs/detections/how-to-query-alert-indices.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[how-to-query-alert-indices]]
+[role="xpack"]
+== How to query alert indices
+This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. (For more information about alert event field definitions, review the [Alert schema].)
+
+The naming conventions for alert indices and their aliases differ depending on which version of {stack} you're running (<8.0 vs 8.0+), but there are aliases that you can query regardless of which version you're using:
+
+[discrete]
+=== Best practices
+If you're using a version before 8.0, you should query the following index alias:
+
+* `.siem-signals-<space id>` (Note that there is no dash or wildcard after the space ID).
+For example, to query all spaces: `.siem-signals-*`.
+
+Once you update to 8.0 or higher, you should query the following index alias, which will include both legacy (>8.0) and new alerts:
+
+* `.alerts-security.alerts-<space id>` (Note that there is no dash or wildcard after the space ID).
+For example, to query all spaces: `.alerts-security.alerts-*`.
+
+[discrete]
+=== Acceptable practices
+Queries on the following index aliases will function regardless of your {stack} version, but will not follow the `.alerts` naming convention and may be deprecated in future releases:
+
+* `.siem-signals-<space ID>` (Note that there is no dash or wildcard after the space ID).
+For example, to query all spaces: `.siem-signals-*`.
+
+To learn more about how alert indices and aliases are defined in different versions of the {stack}, see below.
```

Comment:
Done, with the result that the first section is bigger. For the record my thinking here was that the page should tell users what to do as quickly as possible, before sharing additional info. I can't say I prefer the result but I hope you all disagree!

Reply:
@benironside Ha, yeah I actually kind of agree, it could still foreground the main points a bit more quickly. I'll take a stab at it on my local machine and share if I come up with any more suggestions